deploy/examples/applicationmonitoring_v1alpha1_blackboxtarget_cr.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: applicationmonitoring.integreatly.org/v1alpha1
```

Comment:
@davidkirwan same as other file, probably best to call this `BlackboxTarget.yaml`